src/include/meta.h

```diff
@@ -76,6 +76,9 @@ struct __wt_ckpt {
 
 	uint64_t write_gen;			/* Write generation */
 
+						/* Timestamps */
+	wt_timestamp_t oldest_start_ts, newest_start_ts, newest_stop_ts;
+
```

Comment:
Why have these all on the same line?

Reply:
The addresses are a triplet, that is, there shouldn't ever be one or two of them alone (in previous incarnations of the code it was actually a structure). That's why putting them all on the same line made sense to me.

Let me know if you feel strongly.

Reply:
I totally did not understand why they were on the same line, so that visual did not make sense to me. I don't think we do that visual anywhere else, do we? They're already grouped by whitespace. I think having them on separate lines and adding a comment before the group saying `/* These are a triplet, all or none should always appear. */ is more clear (to me anyway).